tools/convert/rs-convert.cpp

```diff
@@ -52,37 +62,63 @@ int main(int argc, char** argv) try
         : switchColor.isSet() ? rs2_stream::RS2_STREAM_COLOR
         : rs2_stream::RS2_STREAM_ANY;
 
-    if (outputFilenameCsv.isSet()) {
+    if (outputFilenameCsv.isSet())
+    {
         converters.push_back(
             make_shared<rs2::tools::converter::converter_csv>(
                 outputFilenameCsv.getValue()
                 , streamType));
     }
 
-    if (outputFilenamePng.isSet()) {
+    if (outputFilenamePng.isSet())
+    {
         converters.push_back(
             make_shared<rs2::tools::converter::converter_png>(
                 outputFilenamePng.getValue()
                 , streamType));
     }
 
-    if (outputFilenameRaw.isSet()) {
+    if (outputFilenameRaw.isSet())
+    {
         converters.push_back(
             make_shared<rs2::tools::converter::converter_raw>(
                 outputFilenameRaw.getValue()
                 , streamType));
     }
 
-    if (outputFilenameBin.isSet()) {
+    if (outputFilenameBin.isSet())
+    {
         converters.push_back(
             make_shared<rs2::tools::converter::converter_bin>(
                 outputFilenameBin.getValue()));
     }
 
-    if (converters.empty() && !outputFilenamePly.isSet()) {
+    if (converters.empty() && !outputFilenamePly.isSet())
+    {
         throw runtime_error("output not defined");
     }
 
+    unsigned long long first_frame = 0;
+    unsigned long long last_frame = 0;
+    uint64_t start_time = 0;
+    uint64_t end_time = 0;
+
+    if (frameNumberStart.isSet())
+    {
+        first_frame = stoi(frameNumberStart.getValue());
+    }
+    if (frameNumberEnd.isSet())
+    {
+        last_frame = stoi(frameNumberEnd.getValue());
+    }
+    if (StartTime.isSet())
+    {
+        start_time = SECTONSEC *(atof(StartTime.getValue().c_str())); //seconds to nanoseconds 
```

Comment:
`std::strtod` is better than `atof`.
But there's also an implicit type conversion here: from double to uint64_t, which can cause warnings: please make sure no warnings are issued when compiling this file.